packages/expo-contacts/android/src/main/java/expo/modules/contacts/next/domain/model/contact/NewContact.kt

```diff
@@ -0,0 +1,19 @@
+package expo.modules.contacts.next.domain.model.contact
+
+import android.content.ContentProviderOperation
+import android.provider.ContactsContract
+import expo.modules.contacts.next.domain.model.Insertable
+
+data class NewContact(val modelsToInsert: List<Insertable>) {
+  fun toInsertOperations(): ArrayList<ContentProviderOperation> {
+    val output = arrayListOf<ContentProviderOperation>()
```

Comment:
```suggestion
  fun toInsertOperations(): List<ContentProviderOperation> {
    val output = mutableListOf<ContentProviderOperation>()
```
If you have to use `ArrayList` here, I'm okay with it. However, always prefer to use `listOf`. Also, you don't need to use `mutableListOf` in this case. I would say that the `buildList` function is a better choice here - https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.collections/build-list.html